src/core/agent/services/identifierService.ts

```diff
@@ -140,35 +150,116 @@ class IdentifierService extends AgentService {
     };
   }
 
+  async resolvePendingIdentifier() {
+    const pendingIdentifierCreation = await this.basicStorage.findById(
+      MiscRecordId.IDENTIFIERS_PENDING_CREATION
+    );
+
+    if (
+      pendingIdentifierCreation &&
+      Array.isArray(pendingIdentifierCreation?.content?.name)
+    ) {
+      for (const name of pendingIdentifierCreation.content.name) {
+        let metadata: Omit<IdentifierMetadataRecordProps, "id" | "createdAt">;
+        const [themeString, rest] = name.split(":");
+        if (!rest) {
+          throw new Error(
+            `${IdentifierService.IDENTIFIER_METADATA_RECORD_MISSING}`
```

Comment:
How does this error make any sense - identifier metadata record missing?

Just assume it's in the right format.

Reply:
@Sotatek-BaoHoanga Just assume it's in the right format, means remove this `if (!rest)` check and the error.